main_test.go

```diff
@@ -41,17 +38,10 @@ func init() {
 }
 
 func binaryPath(t *testing.T) string {
-	switch runtime.GOOS {
-	case "darwin":
-		return binaryPathDarwin
-	case "linux":
-		return binaryPathLinux
-	case "windows":
-		return binaryPathWindows
-	default:
-		t.Fatalf("OS %q not supported", runtime.GOOS)
-		return ""
+	if runtime.GOOS == "windows" {
```

Comment:
The switch statement should be changed to the supported os and archs. Currently we don't support `aix`, `android`, `dragonfly`, `ios`, `js`,  `plan9` and `solaris`.

Reply:
> The switch statement should be changed to the supported OSs. Currently we don't support `aix`, `android`, `dragonfly`, `ios`, `js`, `plan9` and `solaris`.

@gandarez Or shall I use this?

```go

	var filename string
	switch runtime.GOOS {
	case "darwin":
		fallthrough
	case "linux":
		fallthrough
	case "windows":
		filename = fmt.Sprintf("./build/wakatime-cli-%s-%s", runtime.GOOS, runtime.GOARCH)
	default:
		t.Fatalf("OS %q not supported", runtime.GOOS)
	}
	
	if runtime.GOOS == "windows" {
		filename += ".exe"
	}

	return filename
```

Reply:
I'd prefer to have early returns:
```Go
func binaryPath(t *testing.T) string {
	filename := fmt.Sprintf("./build/wakatime-cli-%s-%s", runtime.GOOS, runtime.GOARCH)

	switch runtime.GOOS {
	case "darwin", "linux", "freebsd", "netbsd", "openbsd":
		return filename
	case "windows":
		return filename + ".exe"
	default:
		t.Fatalf("OS %q not supported", runtime.GOOS)
		return ""
	}
}
```

Reply:
I love early returns too. Makes code more readable.